symex/Cargo.toml

```
@@ -43,8 +43,10 @@ paste = "1.0.14"
 [features]
 # Solvers
 bitwuzla = ["dep:bitwuzla"]
+bitwuzla-vendor-cadical = ["dep:bitwuzla", "bitwuzla/vendor-cadical"]
```

See previous comment
```suggestion
build-bitwuzla = ["dep:bitwuzla", "bitwuzla/vendor-cadical"]
```